notification2-examples/hello-world-microservice/src/main/java/c8y/example/notification/microservice/NotificationExample.java

```diff
@@ -188,6 +190,20 @@ private Optional<TenantProperties> getTenantProperties() {
                 .build());
     }
 
+    private void unsubscribe() {
+        if (token != null) {
+            // To unsubscribe, you can pass any token to the unsubscribe API. It does not have to be the one first obtained. You can generate a fresh token
```

Comment:
question: Can we unsubscribe with an expired token? It would be good to be able to. 

We each/all start finding new questions about our own API and behavior as we do these reviews :-)

Reply:
I will just add a comment you can :)

Reply:
Done.